packages/components/src/site-thumbnail/use-mshots.tsx

```diff
@@ -0,0 +1,120 @@
+import { addQueryArgs } from '@wordpress/url';
```

Comment:
Does it make sense to rename `use-mshots.tsx` to `use-mshots-img.tsx`, to more closely match the primary component name?